ports/espressif/common-hal/dotclockframebuffer/DotClockFramebuffer.c

```diff
@@ -114,7 +114,7 @@ void common_hal_dotclockframebuffer_framebuffer_construct(dotclockframebuffer_fr
 
     esp_lcd_rgb_panel_config_t *cfg = &self->panel_config;
     cfg->timings.pclk_hz = frequency;
-    cfg->timings.h_res = width + overscan_left;
+    cfg->timings.h_res = (width + overscan_left + 15) / 16 * 16; // round up to multiple of 16
```

Comment:
```suggestion
    cfg->timings.h_res = int((width + overscan_left + 15) / 16) * 16; // round up to multiple of 16
```

Reply:
It's still giving the same error. I think without the int(), it just re-multiplies.

Reply:
Tried from the start and it worked this time, so I must have uploaded the wrong firmware.